test/helm3/ct.yaml

```diff
@@ -0,0 +1,20 @@
+debug: true
+target-branch: master
+chart-dirs:
+  - stable
+  - staging
+excluded-charts:
+  - common
+  - dex-controller # Moved to a different helm repo
+chart-repos:
+  - mesosphere-staging=https://mesosphere.github.io/charts/staging
+  - mesosphere-stable=https://mesosphere.github.io/charts/stable
+  - kubefed-charts=https://raw.githubusercontent.com/kubernetes-sigs/kubefed/master/charts
+  - jetstack-charts=https://charts.jetstack.io
+  - banzaicloud=https://kubernetes-charts.banzaicloud.com
+  - helm-stable=https://kubernetes-charts.storage.googleapis.com/
+  - dex-controller=https://mesosphere.github.io/dex-controller/charts
+  - kommander=https://mesosphere.github.io/kommander/charts
+  - kubecost=https://kubecost.github.io/cost-analyzer
+  - yakcl=https://mesosphere.github.io/yakcl/charts
+helm-extra-args: --timeout 600s
```

Comment:
Is 600s a bit high? Do we know what the longest average run is?

Reply:
default is 5ms, but yeah i have seen a few things run for around 10m in helm2 land.

Reply:
ok :+1: